Imgui/src/ImGuiImplSDL2.cpp

```diff
@@ -71,25 +71,57 @@ ImGuiImplSDL::ImGuiImplSDL(const ImGuiDiligentCreateInfo& CI,
     }
 }
 
-ImGuiImplSDL::~ImGuiImplSDL() { ImGui_ImplSDL2_Shutdown(); }
+ImGuiImplSDL2::~ImGuiImplSDL2() { ImGui_ImplSDL2_Shutdown(); }
 
-void ImGuiImplSDL::NewFrame(Uint32            RenderSurfaceWidth,
-                            Uint32            RenderSurfaceHeight,
-                            SURFACE_TRANSFORM SurfacePreTransform)
+void ImGuiImplSDL2::NewFrame(Uint32            RenderSurfaceWidth,
+                             Uint32            RenderSurfaceHeight,
+                             SURFACE_TRANSFORM SurfacePreTransform)
 {
     ImGui_ImplSDL2_NewFrame();
     ImGuiImplDiligent::NewFrame(RenderSurfaceWidth, RenderSurfaceHeight,
                                 SurfacePreTransform);
 }
 
-void ImGuiImplSDL::Render(IDeviceContext* pCtx)
+void ImGuiImplSDL2::Render(IDeviceContext* pCtx)
 {
     ImGuiImplDiligent::Render(pCtx);
 }
 
-bool ImGuiImplSDL::HandleSDLEvent(const SDL_Event* ev)
+bool ImGuiImplSDL2::HandleSDLEvent(const SDL_Event* ev)
 {
     return ImGui_ImplSDL2_ProcessEvent(ev);
 }
 
+float ImGuiImplSDL2::GetContentScaleForWindow(SDL_Window* pWindow)
+{
+    return ImGui_ImplSDL2_GetContentScaleForWindow(pWindow);
+}
+
+float ImGuiImplSDL2::GetContentScaleForDisplay(int pDisplayIndex)
+{
+    return ImGui_ImplSDL2_GetContentScaleForDisplay(pDisplayIndex);
+}
+
+void ImGuiImplSDL2::SetGamepadMode(GAMEPAD_MODE pGamepadMode, _SDL_GameController** pManualGamepadsArray, int pManualGamepadsCount)
+{
+    ImGui_ImplSDL2_GamepadMode GamepadMode{};
+    switch (pGamepadMode)
+    {
+        case GAMEPAD_MODE_AUTO_FIRST:
+        {
```

Comment:
Let's not use braces here - they don't add to readability. Besides, last case doesn't use them.